### ISSUE TITLE
Explore making the root font size 18px.

### ISSUE THREAD

**Issue by wboucherfischer** [mozilla/protocol#417]
Explore what it would take to make the default paragraph text font size 18px for better accessibility, readability and improve the color contrast ratios for normal/bold to make more flexible.

**Comment by stephaniehobson** [comment]
I would rather improve the colour contrast of the font than change the text size. Having a different base font size for Mozilla and Firefox themes will cause all kinds of problems.

**Comment by wboucherfischer** [comment]
Why can't we make 18px the base font size for both? It makes it easier from
a color contrast perspective because you can use normal.  Also it's bigger
and more readable across all devices and bps.

**Comment by craigcook** [comment]
We can increase the root size for all themes, though that's a sweeping change that we should approach carefully. 16px is the default in browsers so it's a good standard. 

16px is also "Body MD" in the type scale and I feel like anything called 'medium' should be the norm, so does this mean adjusting the entire type scale around this new root? It's already been suggested to add a Body XL size but maybe that becomes moot if we just embiggen all the body sizes.

**Comment by wboucherfischer** [comment]
Then we'd have to change everything on headings, sizing etc.  It would need
more discussions.

**Comment by johngruen** [comment]
I think visual designers on both teams are looking at a 4px baseline grid with font-sizes and line-heights at multiples thereof.

AFAICT, there's no specific WCAG target font size for body fonts, but 16 px does seem to be a [rule](https://uxtricks.design/blogs/ux-design/accessibility-standards/) of [thumb](https://accessibleweb.com/wcag_2_0_developers/minimum-font-size/). The other A11y rule would be to verify that zooms up to 200% will still render correctly. 

Recommend we close this issue b/c changing the base-font will precipitate a raft of downstream changes.

**Comment by wboucherfischer** [comment]
There are no WCAG target font sizes, but 18px is considered large size, so there's a 3:1 color contrast ratio vs. 4.5:1 for other text below 18px. 18px is also easier to read across a wide variety of mobile and tablet devices, and higher resolution monitors. It's also easier to read from an accessibility standpoint for older people or people with minor far-sight problems who don't wear reading glasses. I'm fine if we close it, I would like to explore this at some point in the future.

**Comment by johngruen** [comment]
so my concern here is that this change would have cascading effects that would mean that we need to rethink relationships across the site. it's worth discussion, but i am kicking this from this sprint b/c is it not actionable.

**Comment by craigcook** [comment]
We explored it. Decided not to.
